index.html

```diff
@@ -60,7 +60,7 @@ <h2 class="h2-dark">
 				</h2>
 				<div class="card-1">
 					<div class="img">
-						<img src="Works-Images/Img Placeholder.png" class="poster">
+						<img src="Works-Images/Img Placeholder.png" class="poster" alt="">
```

Comment:
- [ ] Please always try to add the `alt` tag text in your *img* tag. You can imagine a scenario where a user's internet is very slow and not able to load the media of the website, then in that kind of situation, this **alt** tag will help the user to understand what the media is all about. Just consider my *Note* in the previous comment.

**Point to be noted:** *Sometimes we use the image as part of the decoration as secondary information best example I can give you is icons etc then alt text is not needed there but as of right now you are learning to make a habit to use alt text everywhere.*